mypyc/test-data/run-dicts.test

```diff
@@ -91,6 +91,10 @@ od.move_to_end(1)
 assert get_content(od) == ([3, 1], [4, 2], [(3, 4), (1, 2)])
 assert get_content_set({1: 2}) == ({1}, {2}, {(1, 2)})
 assert get_content_set(od) == ({1, 3}, {2, 4}, {(1, 2), (3, 4)})
+
+d = {'a': 1, 'b': 2}
+d.clear()
+assert d == {}
```

Comment:
It would be nice to also test this with a `DefaultDict` instance (still using `Dict[...]` as the declared type) to test the other code path.

Reply:
See my most recent comments in #9721 -- they also apply here.

Reply:
At least the lines that call `clear()` need to be moved away from `driver.py`, since this file doesn't get compiled.